docs/CAPS/Introduction.md

```diff
@@ -41,7 +41,7 @@ Several differences exist between the BIDS and CAPS specifications.
 
 - CAPS assumes that the session is always present in a BIDS dataset even though there is a single session. In other words, all datasets are considered longitudinal, even when they have only one session.
 
-- NifTi files generated by a Clinica pipeline are always compressed, while this is not systematic with the BIDS specifications.
+- NIfTI files generated by a Clinica pipeline are always compressed, while this is only recommended by the BIDS specifications.
```

Comment:
je ne comprend pas la phrase "while this is only recommended ..", est-ce que ce ne serais pas plutôt "[..] compressed, this is recommended by .."

Reply:
He wants to say Clinica forces compression of NIfTI files while this is only a recommendation in the BIDS spec.

Reply:
"systematic" is not idiomatic English and reads as word-to-word translated from French

Reply:
Tbh we could drop this part of the sentence entirely.

Reply:
Granted for "systematic"  but I still think the phrasing is a bit awkward. I would change it to something of the sort "[..] are always compressed. Compression is  "recommended" but not compulsory*  in the BIDS specifications" 

*or the appropriate BIDS term

Reply:
> Tbh we could drop this part of the sentence entirely.

Also works for me